static/src/javascripts/projects/common/modules/video/youtube-player.js

```diff
@@ -17,7 +17,7 @@ define([
 
     function loadYoutubeJs() {
         fastdom.write(function () {
-            if (!document.querySelector(scriptId)) {
+            if (!document.getElementById(scriptId)) {
```

Comment:
This is a bug fix, scriptId is a string but doesn't include the '#' so this condition was always true with querySelector.